data/language/de-DE.yml

```diff
@@ -116,7 +116,9 @@ strings:
   102: Objekt im Wege
   103: Spiel laden
   104: Spiel speichern
+  # TODO: requires updating. English now reads: 'Quit to menu'
```

Comment:
Did you miss similar change in sk-SK.yml file?

Reply:
Good spotting. Apparently I missed it.
Once I'm back at my PC I'll change it.
Thank You!

Reply:
Fixed.